File: React/HandsOn/react-functional-components/src/Product.js
const Product = ({ item }) => {
  return (
    <>
      <tr>
        <td>{item.id}</td>
        <td>{item.name}</td>
        <td>{item.price}</td>
        <td>{item.stock}</td>
      </tr>
    </>
  );
};
const Cart = () => {
  return (
    <div className="container">
      <table className="table table-striped">
        <thead className="bg-primary">
          <th>Id</th>
          <th>Name</th>
          <th>Price</th>
          <th>Stock</th>
        </thead>
        <tbody>
          <Product item={{ id: 1, name: "mouse", price: 300, stock: 10 }} />
          <Product item={{ id: 2, name: "keyboar", price: 300, stock: 10 }} />
          <Product item={{ id: 3, name: "bottle", price: 200, stock: 10 }} />
          <Product item={{ id: 4, name: "bisucte", price: 30, stock: 10 }} />
        </tbody>
      </table>
    </div>
  );
};
export default Cart;

File: API/HandsOn/HandsOnAPIWIthEF/Entities/Product.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace HandsOnAPIWIthEF.Entities
{
    public class Product
    {
        [Key] //set as primary key
        public int Id { get; set; }
        [Required] //set as not null
        public string Name { get; set; }
        public int? Price { get; set; } //set null constraint
        public int? Stock { get;set; }
    }
}
